documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,50 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+* This step ApiKeyValueMapUpload helps you to store one or more key value pairs of data in a grouping called a map, or key value map.
```

Comment:
Either "This step stores one or ..." or "ApiKeyValueMapUpload stores one or ..."

Reply:
fixed